pygmt/helpers/decorators.py

```diff
@@ -224,6 +226,18 @@ def fmt_docstring(module_func):
             aliases.append("- {} = {}".format(arg, alias))
         filler_text["aliases"] = "\n".join(aliases)
 
+    filler_text["table-like"] = (
+        "numpy.ndarray or "
+        "pandas.DataFrame or "
+        "xarray.Dataset"
+        # "geopandas.GeoDataFrame"
+    )
```

Comment:
I'm OK with the `table-like`, but wondering if we could rewrite it to:
```
filler_text["table-like"] = " or ".join(["numpy.ndarray", "pandas.DataFrame", "xarray.Dataset"])
```

Reply:
Good idea! I've done the change at e9a798b6c932cf6134b37c46f4e397ec1e7211a6 and incorporated the other suggested changes too.